dev-guide/develop.md

```diff
@@ -0,0 +1,68 @@
+# Build for Development
+
+If you want to develop the TiDB project, you can follow this guide.
+
+Before you begin, check the [supported platforms](./requirements.md#supported-platforms) and [prerequisites](./requirements.md#prerequisites) first.
+
+## Build TiKV
+
++ Get TiKV source code from GitHub
+
+    ```bash
+    git clone https://github.com/tikv/tikv.git
+    cd tikv
+    ```
+
++ Run all unit tests:
+
+    ```bash
+    make test
+    ```
+
++ Build in release mode:
+
+    ```bash
+    make release
```

Comment:
What does development mean? Does it mean develop TiKV and send a PR or just start a TiKV instance and develop a client to connect to it? The former doesn't require release, which is slow to compile.

Reply:
@BusyJay What's your suggested version?

Reply:
> What does development mean? Does it mean develop TiKV and send a PR or just start a TiKV instance and develop a client to connect to it? The former doesn't require release, which is slow to compile.

@BusyJay Any suggested version? Thanks~